### ISSUE TITLE
The Dutch internationalization is different for The Netherlands 'nl-nl' than for Belgium 'nl-be'

### ISSUE THREAD

**Issue by deidigitalis** [moment/moment#3410]
**Description of the Issue and Steps to Reproduce:**

Using the localization for Belgium is wrong. In Belgium the separator symbol is '/' and in The Netherlands is '-'.

The current localization 'nl' should be 'nl-nl' and it is needed another one 'nl-be' with the following change:

```
longDateFormat : {
            LT : 'HH:mm',
            LTS : 'HH:mm:ss',
            L : 'DD/MM/YYYY',
            LL : 'D MMMM YYYY',
            LLL : 'D MMMM YYYY HH:mm',
            LLLL : 'dddd D MMMM YYYY HH:mm'
        }
```

Best regards.

**Comment by kevinzwh** [comment]
Taking a stab at this issue, will have a PR open for it in a moment!

**Comment by icambron** [comment]
Closed by #3416
